docs/sections/data/groups.md

```diff
@@ -71,54 +67,44 @@ PK    Type         Created
  380  CalcJobNode  2078D:17h:46m ago
 1273  CalcJobNode  2078D:18h:03m ago
 ...
-
 ```
 
-Conversely, if you want to see all the groups a node belongs to, you can run:
+Conversely, if you want to see all the groups a node belongs to, you can use its PK and run:
 
 ```{code-block} console
-
 $ verdi group list -a -A --node 380
 PK    Label            Type string    User
 ----  ---------------  -------------  ---------------
    1  tutorial_pbesol  core           aiida@localhost
    7  20200705-071658  core.import    aiida@localhost
-
 ```
 
 ### Creating and manipulating groups
 
 Lets make a new group:
 
 ```{code-block} console
-
 $ verdi group create a_group
 Success: Group created with PK = 8 and name 'a_group'
-
 ```
 
-If we want to change the name of the group at any time:
+If you want to change the name of the group at any time try:
 
 ```{code-block} console
-
 $ verdi group relabel a_group my_group
 Success: Label changed to my_group
-
 ```
 
-Now we can add one or more nodes to it:
+Now you can add one or more nodes to it by listing any number of node PKs:
```

Comment:
Lot's of optional-sounding "can" phrases are used here. No big deal, but maybe we can also use the _imperative mood_ to give instructions instead? (see the examples in #360 ). The "can" suggestions in the React tutorial are _really_ suggestions, but here it feels more like we're giving instructions. The sentence would become:

```suggestion
Add one or more nodes to your new group using node PKs from the `tutorial_pbesol` group we inspected earlier:
```